ponyup-init.sh

```diff
@@ -2,6 +2,7 @@
 
 set -o errexit
 set -o nounset
+set -o pipefail
```

Comment:
This isn't a valid option for Posix shell support. We are keeping this file working with sh support. You'll need to remove. You'll see CI failed after the addition of this.